src/build_database.py

```
@@ -1,587 +1,587 @@
-if __name__ == "__main__":
+"""
+競馬データ収集ツール
+
+netkeiba.comから指定された期間のレース情報を取得し、データベースに保存するツール。
+このモジュールは、スクレイピング機能とデータベース操作を組み合わせたデータ収集パイプラインを提供します。
+"""
+
+import argparse
+import re
+import time
+from datetime import datetime, date
```

Ruff (0.11.9)

`datetime.date` imported but unused

Remove unused import: `datetime.date`

(F401)